github/ContentFile.py

```diff
@@ -181,6 +193,7 @@ def _initAttributes(self):
         self._download_url = github.GithubObject.NotSet
         self._git_url = github.GithubObject.NotSet
         self._html_url = github.GithubObject.NotSet
+        self._last_modified = github.GithubObject.NotSet
```

Comment:
I don't think it's needed here.

Reply:
Agreed. since i am dropped the check above, this is not required.